packages/react/src/search-async/index.tsx

```diff
@@ -1,2 +1 @@
 export * from "./SearchAsync";
-export type { ActionMeta } from "chakra-react-select";
```

Comment:
Å fjerne ting konsumenter potensielt bruker er strengt tatt en breaking change, i så fall ville det vel blitt en ny major versjon

Reply:
eksporterer den fortsatt fra `SearchAsync.tsx`, er bare flyttet på så man eksporterte alle typene fra chakra-react-select på ett sted

Reply:
nevermind we back siden jeg ikke exporter SelectInstance og GroupBase